nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -173,6 +175,29 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
```

Comment:
This property only applies to JSON_STANDARD.

- Is there a reason it can't apply to extended?
- If you use extended what do you get?
- That should be in the description I think.
- Do we want to validate that you don't set DATE_FORMAT with JSON_EXTENDED?

Reply:
If they have them both set, it's not a problem. The processor just ignores the date format configuration.

Reply:
Right, the code will work.  The user however may be a bit surprised.  "Why let me configure it, and then ignore what I put in without telling me?"

Reply:
would you accept a PR on your ... PR?

Reply:
Sure.

Reply:
Actually, this get's into the weeds if you are going to keep it required. 
Here is another idea - Can we rename the Display Name of the property to "Date Format For Standard JSON" ?

Reply:
Also, some of the IT tests fail because of pretty printing causes comparison failures.

Reply:
I didn't see any failures when I ran. What are you seeing?

Reply:
When I run the tests in intellij ( I did do a PR, but there was too much shrapnel ), the tests that compare the content to the known values fail, because the json is pretty printed in the result but not the expected.

Reply:
Just rebased and reran the tests in IntelliJ and got 52/52 for the entire processor test package. Don't know what to tell ya.